x-pack/plugins/apm/server/lib/services/get_service_transaction_groups_metrics/index.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Coordinate } from '../../../../typings/timeseries';
+import { EVENT_OUTCOME } from '../../../../common/elasticsearch_fieldnames';
+import { LatencyAggregationType } from '../../../../common/latency_aggregation_types';
+import { getLatencyValue } from '../../helpers/latency_aggregation_type';
+import { Setup, SetupTimeRange } from '../../helpers/setup_request';
+import { getTimeseriesDataForTransactionGroups } from '../get_service_transaction_groups/get_timeseries_data_for_transaction_groups';
+
+export async function getServiceTransactionGroupsMetrics({
```

Comment:
IIRC we wanted to avoid using metrics as it's an overloaded term?

Reply:
did we come up with an acceptable term? I don't recall

Reply:
I'm using statistics, so in this case to distinguish maybe something like primary statistics/comparison statistics. Perhaps @sqren has better ideas.

Reply:
Hmm... I never fell in love with "statistics" as a term but I don't have anything much better.
I do agree that metrics is overloaded (I might have been the one complaining about it in the first place). 

Would this be terrible?
```suggestion
export async function getServiceTransactionGroupsAggregationValues({
```

or

```suggestion
export async function getServiceTransactionGroupsAggValues({
```

or

```suggestion
export async function getServiceTransactionGroupsAggResults({
```